package.json

```
@@ -13,7 +13,7 @@
   "dependencies": {
     "@netlify/edge-functions": "^2.3.1",
     "@netlify/functions": "^2.6.0",
-    "@netlify/remix-adapter": "^2.3.0",
+    "@netlify/remix-adapter": "^2.3.1",
     "@netlify/remix-edge-adapter": "^3.2.0",
```

```suggestion
    "@netlify/remix-edge-adapter": "^3.2.1",
```

This was released (and fixed) as well in https://github.com/netlify/remix-compute/pull/281